shell/platform/android/io/flutter/embedding/engine/systemchannels/SpellCheckChannel.java

```diff
@@ -25,13 +25,13 @@
  * fetch spell check results for the specified text.
  *
  * <p>Once the spell check results are received by the {@link
- * io.flutter.plugin.editing.SpellCheckPlugin}, it will send to the framework the message {@code
- * SpellCheck.updateSpellCheckResults} with the {@code ArrayList<String>} of encoded spell check
- * results (see {@link
- * io.flutter.plugin.editing.SpellCheckPlugin#onGetSentenceSuggestions(SentenceSuggestionsInfo[])}
- * for details) with the text that these results correspond to appeneded to the front as an
- * argument. For example, the argument may look like: {@code {"Hello, wrold!",
- * "7.11.world\nword\nold"}}.
+ * io.flutter.plugin.editing.SpellCheckPlugin}, it will send back to the framework the {@code
+ * ArrayList<String>} of encoded spell check results (see {@link
+ * io.flutter.plugin.editing.SpellCheckPlugin#onGetSentenceSuggestions} for details) with the text
+ * that these results correspond to appended to the front as an argument. For example, the argument
+ * may look like: {@code {"Hello, wrold!", "7.11.world\nword\nold"}}. Please note that the {@link
```

Comment:
per our style guide you can remove "Please note that", since it doesn't add anything.